libcontainer/integration/exec_test.go

```diff
@@ -905,3 +905,47 @@ func TestMountCgroupRW(t *testing.T) {
 		}
 	}
 }
+
+func TestPrestartHook(t *testing.T) {
+	if testing.Short() {
+		return
+	}
+	root, err := newTestRoot()
+	ok(t, err)
+	defer os.RemoveAll(root)
+
+	rootfs, err := newRootfs()
+	ok(t, err)
+	defer remove(rootfs)
+
+	config := newTemplateConfig(rootfs)
+	pwd, _ := os.Getwd()
+
+	hookPath := filepath.Join(pwd, "../../script", "hook.py")
+	prestartCmd := configs.Command{Path: hookPath}
```

Comment:
With the “explicit `argv[0]`” approach, does this command freak out without `argv[0]` being set?  We might want a “if `args` isn't declared, use `[Path]`” fallback, which I suggested in [my post to the list][1].

[1]: https://groups.google.com/a/opencontainers.org/forum/#!topic/dev/luaaorsya10